src/news/release-8.0.md

```diff
@@ -0,0 +1,146 @@
+- Don't instantiate nixpkgs in Flake by [@getchoo](https://github.com/getchoo) and [@Scrumplex](https://github.com/Scrumplex) in [#1574](https://github.com/PrismLauncher/PrismLauncher/pull/1574)
+- Don't update Java minimum memory based on maximum memory allocation by [@Trial97](https://github.com/Trial97) in [#1552](https://github.com/PrismLauncher/PrismLauncher/pull/1552)
+- Fix links and add images to modpack changelogs by [@Trial97](https://github.com/Trial97) in [#1370](https://github.com/PrismLauncher/PrismLauncher/pull/1370)
+- Generate predictable UUIDs for offline accounts by [@Scrumplex](https://github.com/Scrumplex) in [#1399](https://github.com/PrismLauncher/PrismLauncher/pull/1399)
+- Implement GLFW/OpenAL override by defining and auto-detecting paths to library files by [@Scrumplex](https://github.com/Scrumplex) in [#1392](https://github.com/PrismLauncher/PrismLauncher/pull/1392)
+- Improve design of instance group header by [@TayouVR](https://github.com/TayouVR) in [#1286](https://github.com/PrismLauncher/PrismLauncher/pull/1286)
+- Improvements and refinements to pack export by [@TheKodeToad](https://github.com/TheKodeToad) in [#1519](https://github.com/PrismLauncher/PrismLauncher/pull/1519)
+- Improvements to modpack UX by [@Trial97](https://github.com/Trial97) in [#1539](https://github.com/PrismLauncher/PrismLauncher/pull/1539)
+- Introduce more strict compiler warnings and fix them by [@getchoo](https://github.com/getchoo), [@Scrumplex](https://github.com/Scrumplex), [@Trial97](https://github.com/Trial97) and [@Ryex](https://github.com/Ryex) in [#1107](https://github.com/PrismLauncher/PrismLauncher/pull/1107), [#1581](https://github.com/PrismLauncher/PrismLauncher/pull/1581) and [#1537](https://github.com/PrismLauncher/PrismLauncher/pull/1537)
+- Refactor mod metadata write by [@Trial97](https://github.com/Trial97) in [#1543](https://github.com/PrismLauncher/PrismLauncher/pull/1543)
+- Reformat codebase by [@TheKodeToad](https://github.com/TheKodeToad) and [@Scrumplex](https://github.com/Scrumplex) in [#1459](https://github.com/PrismLauncher/PrismLauncher/pull/1459)
+- Replace `typedef` with `using` by [@LocalSpook](https://github.com/LocalSpook) in [#1748](https://github.com/PrismLauncher/PrismLauncher/pull/1748)
+- Resolve Forge versions for CurseForge modpacks using meta by [@Trial97](https://github.com/Trial97) in [#1402](https://github.com/PrismLauncher/PrismLauncher/pull/1402)
+- Revamp loader installation UI by [@Scrumplex](https://github.com/Scrumplex) and [@TheKodeToad](https://github.com/TheKodeToad) in [#1230](https://github.com/PrismLauncher/PrismLauncher/pull/1230)
+- Simplify flame and modrinth exports by [@Trial97](https://github.com/Trial97) in [#1391](https://github.com/PrismLauncher/PrismLauncher/pull/1391)
+Commit Commit(sha="f3c089792a5342eeff93f032dca134fe371db4b3") from Skip folders when updating mods (#1643) does not have an author
```

Comment:
I guess this line is here by mistake. This is an incomplete copy of the next line.